rush-plugins/rush-serve-plugin/src/schemas/rush-serve-plugin-options.schema.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Configuration for the @rushstack/rush-serve-plugin.",
+  "description": "For use with the Rush tool, this file provides per-project configuration options. See http://rushjs.io for details.",
+
+  "type": "object",
+  "additionalProperties": false,
+  "required": ["commands"],
+  "properties": {
+    "$schema": {
+      "description": "Part of the JSON Schema standard, this optional keyword declares the URL of the schema that the file conforms to. Editors may download the schema and use it to perform syntax highlighting.",
+      "type": "string"
+    },
+
+    "commands": {
+      "type": "array",
+      "description": "Rush commands for which files should be served. Serving will be skipped if the command is not running in watch mode.",
+      "items": {
+        "type": "string"
+      }
+    },
+
+    "portParameterLongName": {
+      "type": "string",
+      "description": "The name of a custom parameter in command-line.json that provides a port number for the server.",
```

Comment:
Are we concerned that someone may already have a `--port` parameter?

Reply:
Or is the issue here that Rush plugins don't have a way to add new parameters?

Reply:
While being able to specify the port from the command line has some utility, the primary purpose is to allow the auto-assigned port to be forwarded to any operations that need that information, for example if they emit files that contain the local server URL (as our internal repo does). Since this is for forwarding I don't want to make assumptions about the naming convention in this plugin, similar to why the set of command names it is applied to is a plugin config option.